apps/monitoring/version-reporter/helm-charts.yaml

```
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: version-reporter-service-helmcharts
+  namespace: monitoring
+  labels:
+    app: version-reporter-service
+spec:
+  schedule: "*/10 * * * *" # interval would be reviewed after testing complete
+  concurrencyPolicy: "Forbid"
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 300
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      backoffLimit: 3
+      template:
+        spec:
+          restartPolicy: Never
+          serviceAccountName: version-reporter-service-sa
+          containers:
+            - name: version-reporter-service-helmcharts
+              image: sdshmctspublic.azurecr.io/hmcts/version-reporter-services:master-helmcharts #{"$imagepolicy": "flux-system:version-reporter"}
+              imagePullPolicy: Always
+              env:
+                - name: ENVIRONMENT
+                  value: "ptlsbox"
+                - name: CLUSTER_NAME
+                  value: "ss-ptlsbox-00-aks"
+                - name: MAX_VERSIONS_AWAY
+                  value: "2"
+                - name: COSMOS_DB_URI
+                  value: "https://sds-platform-version-reporter.documents.azure.com:443/"
```

Better not to have 1 per platform. We can just have 1 like Jenkins and reuse between both, can be different DBs if u want.
```suggestion
                  value: "https://sds-platform-version-reporter.documents.azure.com:443/"
```

---

That's the plan @adusumillipraveen as per our conversation.